src/views/monitoring/alert/modules/alert-list/AlertDataTable.vue

```diff
@@ -13,10 +13,12 @@
             :select-index.sync="selectIndex"
             :sort-by.sync="sortBy"
             :sort-desc.sync="sortDesc"
```

Comment:
sortBy, sortDesc 처럼, 초기값 대입 이후로, 해당 변수를 이용해 강제로 특정 값을 주입하고자 함이 아니면, 굳이 props 를 변수 바인딩하지 않아도 좋습니다.
```suggestion
            sort-by="created_at"
            :sort-desc="true"
```